src/endpoints/GetRestaurants.js

```diff
@@ -0,0 +1,144 @@
+import http from 'http'
+const port = 8080
+
+http.createServer((req, res) => {
+  //console.log({ headers: req.headers, method: req.method })
+
+  res.setHeader("Access-Control-Allow-Origin", "*")
+  res.setHeader('Access-Control-Request-Method', '*')
+  res.setHeader("Access-Control-Allow-Methods", "OPTIONS, GET")
+  res.setHeader("Access-Control-Max-Age", 2592000)  // 30 days
+  res.setHeader('Access-Control-Allow-Headers', '*')
+
+  if (req.method === "OPTIONS") {
+    res.writeHead(200)
+    res.end()
+    return
+  }
+
+  const cardsData = [
```

Comment:
Please move this data to a separate file called `data.json` and import it here.

Reply:
Fixed